### ISSUE TITLE
Printing hangs on MacOs (WKWebview)

### ISSUE THREAD

**Issue by whalley** [moneymanagerex/moneymanagerex#2831]
MMEX version:
 - [X] 1.3.7
 - [ ] 1.3.6

Operating System:
 - [ ] Windows
 - [X] Mac OSX
 - [ ] Linux 

# Description of the bug

Printing does not work with WkWebView. The latest wxWidgets master branch includes WKWebView in place of WebView on Mac

# Reproduction

Is the bug reproducible? 
 - [X] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Attempt to print from any WebView.

Have raised issue with wxWidgets as it is fully reproducable with the wxWidgets sample app "webview".

**Comment by whalley** [comment]
https://trac.wxwidgets.org/ticket/19063

Will revert code back, and use wxWidgets code base prior to the WebView change for MacOS builds.

**Comment by whalley** [comment]
No code revert to do, have just re-built with earlier wxWidgets release and existing code is still compatible. Note: Need later release that 3.1.4 in order to support MacOS 11.x.

Please assign to me and mark as 'on hold' for now. Needs resolving if we ultimately take 3.1.5.

**Comment by whalley** [comment]
Fast turnaround from the wxWidgets team....

Now fixed - https://github.com/wxWidgets/wxWidgets/commit/7532f93f7e775b732edd0c9b84f7b5c8af8fad74
